tests/h/util/test_logging_filters.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""Test logging filters."""
+from __future__ import unicode_literals
+
+import pytest
+import logging
+from requests.exceptions import ReadTimeout
+from h.util.logging_filters import FilterExceptions
+
+
+class TestLoggingFilters(object):
```

Comment:
I forgot to mention this last time, but if you have a class of tests that test a particular class or function, the convention in our code would be to name it `Test<capitalized class or function name>`.